package.json

```diff
@@ -27,17 +27,25 @@
     "watch-ts": "tsc -w",
     "watch-node": "nodemon dist/app.js",
     "watch": "concurrently -k -p \"[{name}]\" -n \"TypeScript, Node\" -c \"yello.bold, cyan.bold\" \"yarn run watch-ts\" \"yarn run watch-node\"",
-    "docs": "apidoc -i src/ -o public/apidoc/"
+    "docs": "apidoc -i src/ -o public/apidoc/",
+    "lint": "yarn eslint . --ext .js,.jsx,.ts,.tsx"
   },
   "devDependencies": {
     "@types/chai": "^4.1.5",
     "@types/mocha": "^5.2.5",
     "@types/mongoose": "^5.2.17",
     "@types/nock": "^9.3.0",
+    "@typescript-eslint/eslint-plugin": "^4.6.0",
+    "@typescript-eslint/parser": "^4.6.0",
     "chai": "^4.2.0",
+    "eslint": "^7.12.1",
+    "eslint-config-standard": "^16.0.0",
```

Comment:
@jlgasparrini 
a fresh eslint installation and config (follow the eslint guide) these are packages are being included:
Do we have any reason for include others? 

```diff
diff --git a/package.json b/package.json
index b935a40..f00d9f8 100644
--- a/package.json
+++ b/package.json
@@ -34,7 +34,10 @@
     "@types/mocha": "^5.2.5",
     "@types/mongoose": "^5.2.17",
     "@types/nock": "^9.3.0",
+    "@typescript-eslint/eslint-plugin": "^4.6.0",
+    "@typescript-eslint/parser": "^4.6.0",
     "chai": "^4.2.0",
+    "eslint": "^7.12.1",
     "mocha": "^5.2.0",
     "nock": "^10.0.0",
     "ts-node": "^7.0.1",
```

Reply:
shall we remove these as they are not included on a fresh eslint install?
"eslint-plugin-import": "^2.22.1",
"eslint-plugin-node": "^11.1.0",
"eslint-plugin-promise": "^4.2.1",